tests/Fixture/FeaturedTagsFixture.php

```diff
@@ -27,8 +27,8 @@ class FeaturedTagsFixture extends TestFixture
      * @var array
      */
     public array $records = [
-        ['priority' => 1],
-        ['priority' => 2],
-        ['priority' => 3],
+        ['tag_id' => 1, 'priority' => 1],
+        ['tag_id' => 2, 'priority' => 2],
+        ['tag_id' => 3, 'priority' => 3],
```

Comment:
This was the indication it was broken. The `tag_id` column should not be forced to auto-increment just because it's an integer column.